src/state/routing/slice.ts

```diff
@@ -99,61 +35,9 @@ export const routingApi = createApi({
           }
         }
       },
-      async queryFn(args, _api, _extraOptions, fetch) {
-        let fellBack = false
+      async queryFn(args) {
+        const fellBack = false
```

Comment:
Seems like this could be inlined, and doing so would remove a condition entirely (`const method = QuoteMethod.CLIENT_SIDE`), which you could then further inline.